jobbergate-cli/pyproject.toml

```diff
@@ -20,44 +20,43 @@ classifiers = [
 "Changelog" = "https://github.com/omnivector-solutions/jobbergate/blob/main/jobbergate-cli/CHANGELOG.rst"
 
 [tool.poetry.dependencies]
-python = "^3.6.2"
-click = "^8.0.1"
-inquirer = "^2.7.0"
+python = "^3.8"
+click = "^8.1.0"
+inquirer = "^2.10.1"
 PyYAML = "6.*"
-python-dotenv = "^0.19.0"
-sentry-sdk = "^1.4.3"
+python-dotenv = "^1.0.0"
+sentry-sdk = "^1.29.2"
 loguru = "^0.6.0"
 python-jose = "^3.3.0"
-pydantic = "^1.8.2"
-httpx = "^0.22.0"
+pydantic = "^1.10.12"
+httpx = "^0.24.1"
 pendulum = "^2.1.2"
-py-buzz = "^3.1.0"
+py-buzz = "^3.2.0"
```

Comment:
Now that we're dropping python3.6, py-buzz can be upgraded to v4.0.0

Reply:
Now yet:
```
Because plummet (1.1.0) depends on py-buzz (>=3.1.0,<4.0.0)
 and no versions of plummet match >1.1.0,<2.0.0, plummet (>=1.1.0,<2.0.0) requires py-buzz (>=3.1.0,<4.0.0).
So, because jobbergate-cli depends on both py-buzz (^4.0.0) and plummet (^1.1.0), version solving failed.
```